### PR TITLE
Todo List

### PR DESCRIPTION
**Pull request by amejid**
## Milestone: Todo List

In this PR, the following were implemented:-

- No linter errors
- Use the correct Gitflow
- Implemented Add todo function
- Implemented Edit todo function
- Implemented Update todo function

### CODE REVIEW

File: src/pages/About.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+const About = () => <div>Hello from about page</div>;
+export default About;
```
Comment (Said-laasri) [inline src/pages/About.js:2]: - [x] kindly for better design change the location of your text to center maybe

**Review by Said-laasri — APPROVED** [review]
**STATUS: APROVED** :1st_place_medal: 

**Hi @amejid** 

**you have done a great job :+1:** 




 Highlights :100: 
 
**- Your code is clean and dry. :+1:** 
**- You followed the correct git flow. :100:** 
**- Your code follows JavaScript best practices. :1st_place_medal:** 
**- There are no linters errors. :100:** 
**- The readme file is professional. :+1:** 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :100: 

**Optional suggestions**

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code.

- [x] Please Check the comments under the review.

Cheers and Happy coding! :clap:

<hr>
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
<hr>

**Comment by amejid** [comment]
Thank you for reviewing and approving my code.

File: src/pages/About.js (hunk 1 above)
Comment (amejid) [reply src/pages/About.js:2]: Thank you for reviewing my project, I will make the necessary corrections as suggested.

Comment (Said-laasri) [reply src/pages/About.js:2]: @amejid  Great job @amejid  :+1:

**Pull request merged**